core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionFileSystemOutput.java

```diff
@@ -135,11 +135,13 @@ public TransmissionFileSystemOutput(String folderPath) {
     @Override
     public boolean send(Transmission transmission) {
         if (size.get() >= capacityInKB) {
+        	InternalLogger.INSTANCE.logAlways(InternalLogger.LoggingLevel.TRACE, "Persistent storage max capcity has been reached; currently at %s KB. Telemetry will be lost, please set the MaxTransmissionStorageFilesCapacityInMB property in the configuration file.", size.get());
```

Comment:
I think this is a `WARNING` level log. It's an exceptional case, but everything is still operational.

Reply:
I think things are operational, however when this would happen it would indicate loss of telemetry. Imagine a banking application where this occurs and they loss transactional logs which were eventually needed for fraud analysis then this could be critical information. I would keep it as logalways and since this is rare I don't think it would any how effect production loads.

Reply:
I'm suggesting changing it to `InternalLogger.INSTANCE.logAlways(InternalLogger.LoggingLevel.WARN, ...` I also think it should be logAlways.

Reply:
Agreed, WARN makes more sense as it's something that is actionable but not ex